src/FunctionLoader.ts

```diff
@@ -19,6 +19,9 @@ export class FunctionLoader implements IFunctionLoader {
       let scriptFilePath = <string>metadata.scriptFile;
       let script = require(scriptFilePath);
       let userFunction = getEntryPoint(script, metadata.entryPoint);
+      if(!isFunction(userFunction)) {
+        throw "Unable to determine function. Make sure the function has been correctly exported";

```

Comment:
How about `"The resolved entry point is not a function and cannot be invoked by the functions runtime. Make sure the function has been correctly exported."`